registry.json

```diff
@@ -2415,5 +2415,20 @@
     "telegram": "sublayerio",
     "operatorName": "Olivier",
     "github": "keyflight"
+  },
+  {
+    "ticker": "DEN",
+    "poolId": "d28e4ae3848720195f09becd15e6985b54cbf3b62e52bf53c5af8501",
+    "social": {
+      "twitter": "https://twitter.com/denadapool",
+      "facebook": "",
+      "reddit": "https://www.reddit.com/user/DenAdaPool",
+      "instagram": "",
+      "website": "https://denadapool.io",
+      "youtube": "https://www.youtube.com/channel/UCzMsKxzXYKcCtd5cMRQZn9g"
+    },
+    "telegram": "denadapool",
```

Comment:
You need to provide us with a Telegram username, not a group.  We use Telegram to vote on CSPA issues.

Reply:
Sorry about that ..... telegram username is 'aldnns'